app/app.go

```diff
@@ -303,7 +312,16 @@ func NewIxoApp(
 	app.CrisisKeeper = crisiskeeper.NewKeeper(
 		app.GetSubspace(crisistypes.ModuleName), invCheckPeriod, app.BankKeeper, authtypes.FeeCollectorName,
 	)
-	app.UpgradeKeeper = upgradekeeper.NewKeeper(skipUpgradeHeights, keys[upgradetypes.StoreKey], appCodec, homePath)
+
+	// TODO: know what this is.
+	app.FeeGrantKeeper = feegrantkeeper.NewKeeper(appCodec, keys[feegrant.StoreKey], app.AccountKeeper)
```

Comment:
@ig-shaun @demondayza @steveixo 
The Fee Grant Module was added to the Cosmos-SDK. Is there anything that I should be aware of regarding this module in the context of IXO right now?

Reply:
This has ramifications for overall system design, I suggest we have a discussion on this once @steveixo and @ig-shaun are back.